src/app/leaves/actions/actions.ts

```diff
@@ -0,0 +1,27 @@
+"use server";
```

Comment:
Let's update file name. How about we rename out action to `createLeaveRequest`?

Reply:
Fixed